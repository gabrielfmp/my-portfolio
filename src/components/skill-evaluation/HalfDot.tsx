import { Dot, HalfFilledDot } from './styles';
import React from 'react';

const HalfDot = () => (
  <Dot color="#BCBCBC" data-testid="half-dot">
    <HalfFilledDot />
  </Dot>
);

export default HalfDot;
