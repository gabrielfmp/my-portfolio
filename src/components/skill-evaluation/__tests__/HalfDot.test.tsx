import { render } from '@testing-library/react';
import HalfDot from '../HalfDot';
import React from 'react';

test('snapshot', () => {
  const { asFragment } = render(<HalfDot />);

  expect(asFragment()).toMatchSnapshot();
});
