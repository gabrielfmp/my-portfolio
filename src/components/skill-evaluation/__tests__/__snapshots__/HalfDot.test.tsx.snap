// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-5uorsv"
    color="#BCBCBC"
    data-testid="half-dot"
  >
    <div
      class="css-1qe1cwe"
    />
  </div>
</DocumentFragment>
`;
